examples/with-aws-amplify-typescript/pages/index.tsx

```
@@ -140,16 +140,25 @@ const App = (props: Props) => {
     </div>
   )
 }
-App.getInitialProps = async () => {
+
+export const getStaticProps = async () => {
   let result: { data: GetTodoListQuery; errors: {}[] } = await API.graphql(
     graphqlOperation(getTodoList, { id: 'global' })
   )
   if (result.errors) {
     console.log('Failed to fetch todolist. ', result.errors)
-    return { todos: [] }
+    return {
+      props: {
+        todos: [],
+      },
+    }
```

If there are any errors, let's throw instead of returning empty todos